### PR TITLE
Upgrade to latest upstream release 0.7.22

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,10 +8,13 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.bz2
   url: https://github.com/pynac/pynac/releases/download/pynac-{{ version }}/pynac-{{ version }}.tar.bz2
-  sha256: d380312bb6157fe2c0f22bf7af0c5ba1a001ae4ddfaf186e1a240ebfb0ac492e
+  sha256: 7fa8e2a8be77c2a64ac794a5ccb53dcbd232f8a2cccad31b876e90f72fcdc090
+  patches:
+    - memleak.patch
```
Comment (isuruf) [inline recipe/meta.yaml]: I think this can be dropped now.

**Comment by isuruf** [comment]
Can you skip python 3.5 since sage skips 3.5?

**Comment by saraedum** [comment]
I don't understand. Why should I skip a version here just because Sage does?

**Comment by isuruf** [comment]
Last I checked pynac was tied to sage and was not usable outside sage. Has that changed?

**Comment by saraedum** [comment]
I always thought that you could use pynac standalone but it seems you are right: `Pynac is part of Sage and
requires a full Sage installation`

**Pull request merged**